src/lib/__snapshots__/scrape.test.ts.snap

```diff
@@ -0,0 +1,699 @@
+    },
+    Object {
+      "sourceTitle": "Proposition 8: Tracking the Money",
+      "sourceUrl": "http://www.latimes.com/news/local/la-moneymap,0,2198220.htmlstory",
+      "text": "However, he quickly rescinded his statement but fought fire with fire, contributing his star power to raising over $44 million to defeat proposition 8–which passed anyway.",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": "Beware: Celebrity Endorsement Video by Tom Hanks",
+      "sourceUrl": "http://www.myspace.com/video/vid/33546997",
+      "text": "Hanks backed Obama in 2008, posting a video on his Myspace page, explaining his position.",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "In 2012, Hanks forcefully said he would back Obama once again and, despite widespread opinion to the opposite effect, said Obama had exceeded his expectations, saying:",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": "Tom Hanks: I’m Voting for Obama Again",
+      "sourceUrl": "http://www.politico.com/blogs/click/0611/Tom_Hanks_Im_voting_for_Obama_again.html",
+      "text": "If you would have told me a few years ago that 'don't ask, don't tell' would be repealed and about a billion jobs at General Motors and Chrysler would have been saved because the president was smart enough and strong enough and bold enough to do so, I would have said, 'Wow. That's a good president, I think I'll vote for him again.'",
+      "type": "quote",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "He doesn't seem to acknowledge the billions upon billions of taxpayer dollars that were funneled to some of the country's richest men during that bailout, but hey, that's what the PR machine is all about.",
+      "type": "sentence",
+    },
+  ],
+  "notablePerson": Object {
+    "author": "Tom Kershaw",
+    "lastUpdatedOn": "2012-04-05",
+    "name": "Tom Hanks",
+    "relatedPeople": Array [
+      Object {
+        "name": "George Clooney",
+        "slug": "george-clooney",
+      },
+      Object {
+        "name": "Seth MacFarlane",
+        "slug": "seth-macfarlane",
+      },
+      Object {
+        "name": "Al Pacino",
+        "slug": "al-pacino",
+      },
+      Object {
+        "name": "Kevin Smith",
+        "slug": "kevin-smith",
+      },
+    ],
+    "summary": Object {
+      "politicalViews": "Hanks is a hardcore Democrat and liberal, throwing his weight behind gay marriage, alternative energy, environmentalism, and Obama.",
+      "religion": "Hanks has run the gamut of Christian denominations from Mormonism to Catholicism but seems to have settled on Greek Orthodox.",
+    },
+    "tags": Array [
+      "actor",
+      "catholic",
+      "democrat",
+      "director",
+      "greek-orthodox",
+      "liberal",
+      "mormon",
+      "producer",
+      "religious",
+      "white",
+    ],
+  },
+}
+`;
```

Comment:
I think we can definitely work with this format! 👍 

One question: what would happen if a quote has multiple sentences (i.e `p` tags)?

Reply:
I updated the code to push all paragraphs